### PR TITLE
Style: One declaration per line

### PR DESCRIPTION
**Pull request by asmaloney**

### CODE REVIEW

File: CC/src/DgmOctree.cpp [hunk 1]
```diff
@@ -2657,7 +2659,8 @@ ReferenceCloud* DgmOctree::getPointsInCellsWithSortedCellCodes(	cellCodesContain
     unsigned char bitDec2 = (areCodesTruncated ? 0 : bitDec1); //shift for the input codes
 
     cellsContainer::const_iterator p = m_thePointsAndTheirCellCodes.begin();
-    CellCode toExtractCode,currentCode = (p->theCode >> bitDec1); //pred value must be different than the first element's
+    CellCode toExtractCode;
+	CellCode currentCode = (p->theCode >> bitDec1); //pred value must be different than the first element's
```
Comment (dgirardeau) [inline CC/src/DgmOctree.cpp]: alignement?

File: CC/src/DistanceComputationTools.cpp [hunk 2]
```diff
@@ -2111,7 +2121,10 @@ ScalarType DistanceComputationTools::computePoint2TriangleDistance(const CCVecto
     }
     else
     {
-        double tmp0, tmp1, numer, denom;
+        double tmp0;
+		double tmp1;
```
Comment (dgirardeau) [inline CC/src/DistanceComputationTools.cpp]: alignement?

File: CC/src/KdTree.cpp [hunk 3]
```diff
@@ -388,7 +388,9 @@ void KDTree::updateOutsideBoundingBox(KdCell *cell)
 
 ScalarType KDTree::pointToCellSquareDistance(const PointCoordinateType *queryPoint, KdCell *cell)
 {
-    PointCoordinateType dx, dy, dz;
+    PointCoordinateType dx;
+	PointCoordinateType dy;
```
Comment (dgirardeau) [inline CC/src/KdTree.cpp]: alignment?

File: CC/src/KdTree.cpp [hunk 4]
```diff
@@ -414,7 +416,9 @@ void KDTree::pointToCellDistances(	const PointCoordinateType *queryPoint,
 									ScalarType& min,
 									ScalarType& max)
 {
-    PointCoordinateType dx, dy, dz;
+    PointCoordinateType dx;
```
Comment (dgirardeau) [inline CC/src/KdTree.cpp]: alignment?

File: CC/src/KdTree.cpp [hunk 5]
```diff
@@ -425,7 +429,10 @@ void KDTree::pointToCellDistances(	const PointCoordinateType *queryPoint,
 
 ScalarType KDTree::InsidePointToCellDistance(const PointCoordinateType *queryPoint, KdCell *cell)
 {
-    PointCoordinateType dx, dy, dz, max;
+    PointCoordinateType dx;
+	PointCoordinateType dy;
```
Comment (dgirardeau) [inline CC/src/KdTree.cpp]: alignment?

File: CC/src/KdTree.cpp [hunk 6]
```diff
@@ -527,7 +534,8 @@ void KDTree::distanceScanTree(
     KdCell *cell,
     std::vector<unsigned> &localArray)
 {
-    ScalarType min, max;
+    ScalarType min;
+	ScalarType max;
```
Comment (dgirardeau) [inline CC/src/KdTree.cpp]: alignment?

File: CC/src/DgmOctree.cpp (hunk 1 above)
Comment (asmaloney) [reply CC/src/DgmOctree.cpp]: Ah - this method _getPointsInCellsWithSortedCellCodes_ uses spaces where the rest of the file uses tabs. So when I added this with a tab it's misaligned.

It's probably the same for the rest.

**Comment by asmaloney** [comment]
It looks like these files mix tabs and spaces and I just added tabs in methods that were written with spaces...

Let's commit this PR and then I'll fix the spacing in these 3 files.

**Pull request merged**